_startups/ban.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Base Adresse Nationale
+mission: "Un référentiel national ouvert : de l'adresse à la coordonnée géographique."
+domain: adresse.data.gouv.fr
+repository: https://github.com/etalab/ban
```

Comment:
Ok. Considering the lack of a README on `etalab/adresse.data.gouv.fr`, I think the best current destination would be http://adresse.data.gouv.fr/foss/. WDYT?